content/running-tor-browser/contents.lr

```diff
@@ -25,21 +26,20 @@ If you are on a relatively fast connection, but this bar seems to get stuck at a
 
 <img class="col-md-6" src="../../static/images/configure.png" alt="Click 'configure' to adjust network settings.">
 
-If you know that your connection is censored or uses a proxy, you should select this option.
+If you know that your connection is censored or uses a proxy, you should select "Tor Network Settings".
 
 <img class="col-md-6" src="../../static/images/proxy_question.png" alt="Select options for censored connection or proxy use.">
 
 Tor Browser will take you through a series of configuration options.
 
 <img class="col-md-6" src="../../static/images/pluggable-transport.png" alt="Configure Tor bridge options">
 
-The first checkbox asks if access to the Tor network is blocked or censored on your connection.
-If you do not believe this is the case, leave this unchecked.
-If you know your connection is censored, or you have tried and failed to connect to the Tor network and no other solutions have worked, check the checkbox.
+The first checkbox is Quickstart, which allows automatic connection.
```

Comment:
We should explain a little bit more about this option. Like, every time you open Tor Browser, it will try to connect with your previous network settings.

Reply:
'The first checkbox is Quickstart. If you select this option, Tor Browser will attempt to connect to your existing network settings every time you open it.'

Reply:
> 'The first checkbox is Quickstart. If you select this option, Tor Browser will attempt to connect to your existing network settings every time you open it.'

Final commit after this sentence.